### ISSUE TITLE
Non-Inline Volume Control Makes Clicking Volume Button Difficult

### ISSUE THREAD

**Issue by tneaves** [videojs/video.js#2970]
The following javascript yields a vertical volume control; however, on hover, the volume scroll covers the volume button, making the volume button difficult to click (see image, below):

``` javascript
// Configure video.js' control bar used to replace HTML5 controls
options.controlBar = {
    children: [
        "progressControl",
        "playToggle",
    "currentTimeDisplay",
    "timeDivider",
    "durationDisplay",
    "volumeMenuButton",
    "fullscreenToggle"
    ],
    // After declaring children in the array, then specify
    // further options for a child by referencing its name
    volumeMenuButton: { 
        inline: false
    }
```

![image](https://cloud.githubusercontent.com/assets/14412556/12121794/313da290-b3a5-11e5-8962-55b4e2c3176e.png)

**Comment by gkatsev** [comment]
What version of videojs are you using? Also, what browser?
Thanks.

**Comment by tneaves** [comment]
@gkatsev good question. This is in Chrome (but visible across browsers). I am using 5.3.

**Comment by gkatsev** [comment]
Can you try updating to 5.4.6? (It's available on npm and github releases currently. I need to release it on the CDN).

**Comment by tneaves** [comment]
@gkatsev I'll give that a try, later, today, and report back.

**Comment by gkatsev** [comment]
Cool, thanks.

**Comment by tneaves** [comment]
@gkatsev I was able to pull 5.4.4 from npm and it is still bust, there.

**Comment by gkatsev** [comment]
Hm... I'm not able to reproduce it locally. Are you able to put up a reduced test case on a site like jsfiddle.net? Thanks

**Comment by gkatsev** [comment]
This has been fixed in 6.0
